contracts/ColonyNetwork.sol

```diff
@@ -261,6 +266,16 @@ contract ColonyNetwork is ColonyNetworkStorage {
     return skill.children[_childSkillIndex];
   }
 
+  function getChildSkillIndex(uint _skillId, uint _childSkillId) public view returns (uint256) {
```

Comment:
It's a convenience function to make it easier for clients to figure out the reputation proofs. Since it's `view` gas isn't an issue. Also it gets used in this helper function: https://github.com/JoinColony/colonyNetwork/pull/553/files#diff-cfe0340d8acf7ab0683cf395091cd627R703

It's not 100% necessary but I expect it'll be convenient to have around, especially if we want to write more involved domain tests. Having to manually keep track of all the domains and their child skill arrays seems not fun.